src/main/java/com/amashchenko/maven/plugin/gitflow/AbstractGitFlowMojo.java

```diff
@@ -749,8 +762,25 @@ protected void mvnCleanInstall() throws MojoFailureException,
     }
 
     /**
+     * Checks out given {@code tagVersion}, executes mvn deploy & finally
+     * checks out the previous branch again.
+     *
+     * @throws MojoFailureException
+     * @throws CommandLineException
+     */
+    protected void mvnDeploy(String tagVersion) throws MojoFailureException,
+            CommandLineException {
+        getLog().info( String.format( "Deploying tag '%s' to nexus", tagVersion));
```

Comment:
Yeah you're right; it's just that I've started taking 'nexus' as a synonym to mvn repo.